go/service/gregor.go

```diff
@@ -323,6 +323,27 @@ func (g *gregorHandler) ShouldRetryOnConnect(err error) bool {
 	return true
 }
 
+func (g *gregorHandler) forwardMessage(m gregor1.Message) {
+	if g.G().UIRouter == nil {
+		g.Debug("no UI router available when broadcast message arrived")
+		return
+	}
+	ui, err := g.G().UIRouter.GetGregorUI()
+	if err != nil {
+		g.Warning("error fetch a gregor UI router: %s", err)
+		return
+	}
+	if ui == nil {
+		g.Debug("no-op on message, since no Gregor UIs are registerd")
+		return
+	}
+	err = ui.PushMessage(context.Background(), []gregor1.Message{m})
+	if err != nil {
+		g.Warning("Error pushing message to gregor UI: %s", err)
+	}
+	return
+}
```

Comment:
Probably should drop the use of "gregor" in favor of "push" to be consistent with other messages.